### PR TITLE
use fhirserver for db username instead of fhiruser

### PR DESCRIPTION
**Pull request by lmsurpre**
also adjusted fhir-persistence-schema README around a bit

Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

File: fhir-persistence-schema/README.md [hunk 1]
```diff
@@ -1,58 +1,37 @@
...
-
-## Deploy and manage the schema with `com.ibm.fhir.schema.app.Main` 
+## Execute the fhir-persistence-schema command line interface (CLI)
 
 To deploy and manage the schema on a target database, the `fhir-persistence-schema` project includes a Main class that can parallelize the schema updates.
 
-``` shell 
+``` shell
 java -jar ./fhir-persistence-schema-${VERSION}-cli.jar [OPTIONS]
 ```
```
Comment (prb112) [inline fhir-persistence-schema/README.md]:
```suggestion
```

On Windows use `%VERSION%`
```

Comment (prb112) [reply fhir-persistence-schema/README.md]: may also need to specify where the VERSION comes from

Comment (lmsurpre) [reply fhir-persistence-schema/README.md]: I also thought to just make it `*` like we just did in our script...then its platform-neutral, right?

Comment (lmsurpre) [reply fhir-persistence-schema/README.md]: added a note to replace version with the version

**Pull request merged**